tests/backup/backup_helper.go

```diff
@@ -1253,3 +1255,118 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 	}
 	return nil
 }
+
+func CreateRestoreWithCustomResourceType(restoreName string, backupName string, namespaceMapping map[string]string, clusterName string,
```

Comment:
Add a comment for this helper function explaining that it can be used only to deploy custom deployment and it's PVCs and it cannot be used for any other resource type.
In fact we can change the name of the function to something like `CreateCustomRestoreWithPVCs`

Reply:
yes , I've changed the name of function